### PR TITLE
Update leaving-ibm.mdx

### PR DESCRIPTION
**Pull request by ljcarot**
### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/pages/components/leaving-ibm.mdx [hunk 1]
```diff
@@ -20,7 +20,15 @@ import ComponentFeedback from "components/ComponentFeedback";
 
 ## Default
 
-When a use clicks on an external link it will launch the Leaving IBM overlay to help users understand they are leaving IBM.com. It includes a brief message about a possible change in terms, and the host name of the external site is displayed to be transparent.
+Do not apply this notice to all external links. Use this notice only when the user leaves the ibm.com domain to complete a service or transaction (such as commerce or to register for a conference or webinar) on behalf of IBM.  For example, when purchasing a product from IBM Marketplace, a user may choose to use PayPal as a payment method. When they leave ibm.com to go to the PayPal site to complete the transaction, the Leaving IBM overlay would be displayed.
+
+If the destination URL contains parameters, it must be URL encoded to work properly.
+
+The link URL must be preceded with "//www.ibm.com/links/?url="
```
Comment (shixiedesign) [inline src/pages/components/leaving-ibm.mdx:27]: This sentence is showing as a link on the page. Maybe wrap in a code snippet? 
` //www.ibm.com/links/?url=` 

image

**Review by ljcarot — COMMENTED** [review]
Agree, make code snippet

File: src/pages/components/leaving-ibm.mdx (hunk 1 above)
Comment (ljcarot) [reply src/pages/components/leaving-ibm.mdx:27]: @shixiedesign can you make it into code snippet for me?

Comment (JennySanchez) [reply src/pages/components/leaving-ibm.mdx]: @ljcarot Code-snippet update done.

**Review by JennySanchez — APPROVED** [review]
Looks good to me. Thank you, Linda.

**Pull request merged**